openshift-kube-apiserver/filters/deprecatedapirequest/apiaccess_count_controller.go

```diff
@@ -128,7 +129,12 @@ func (c *controller) persistRequestCountForAllResources(ctx context.Context, cur
 			return
 		}
 		for _, arc := range arcs.Items {
-			countsToPersist.Resource(apiNameToResource(arc.Name))
+			gvr, err := apiNameToResource(arc.Name)
```

Comment:
we could try to see if we can add validation to prevent incorrect names.

Reply:
btw which component is responsible for creating `APIRequestCount` resources?

Reply:
Our patched kube-apiserver creates them. I'll look into your suggestion.

Reply:
Done. Added validation via admission plugin.